packages/twenty-front/src/modules/analytics/graphql/queries/track.ts

```diff
@@ -1,8 +1,8 @@
 import { gql } from '@apollo/client';
```

Comment:
putting this comment here as I cannot comment on App.tsx but it seems that there is something wrong with an "empty" modification of App.tsx in this PR:
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/98324f40-5a2d-468e-b988-8ea1c57eed23">